src/main/java/customerproductorder/web/filters/LoginFilter.java

```diff
@@ -0,0 +1,46 @@
+package customerproductorder.web.filters;
+
+import customerproductorder.web.LoginServlet;
+import java.io.IOException;
+import java.util.Enumeration;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LoginFilter implements Filter {
+
+    private ServletContext context;
```

Comment:
is this field is used? Tune your IDE to mark unused code... so you can remove code that is not used.